src/Server/Magento.php

```diff
@@ -36,7 +35,8 @@ class Magento extends Server
     private $verifier;
 
     /**
-     * @param ClientCredentials|array $clientCredentials
+     * @param \League\OAuth1\Client\Credentials\ClientCredentials|array $clientCredentials
+     * @param \League\OAuth1\Client\Signature\SignatureInterface|null   $signature
```

Comment:
Could we simplify the FQN here? I think these are imported at the top so you should be able to reuse that here…

Reply:
* FQN for doc comments
 * Imported namespaces if you're using them in code (class instantiation, static class ...)

Reply:
This is not how I roll sorry. I respect you have an opinion on this but it is not shared amongst all. Please revert. Thanks!